Add recipes for lurk-mode and ob-lurk.

weissjeffm opened the pull request:
### Brief summary of what the package does

A simple emacs mode for the programming language lurk. Also includes another tiny package ob-lurk, which allows org-babel scripts written in lurk to be executed. More sophisticated functionality (like session-aware org-babel execution) is forthcoming.

What is lurk? It's a lisp whose execution is provable in zero knowledge. See http://lurk-lang.org

### Direct link to the package repository

https://github.com/lurk-lang/lurk-emacs

### Your association with the package

Maintainer

### Relevant communications with the upstream package maintainer

None needed 

### Checklist

<!-- Please confirm by replacing `[]` with `[x]`: -->

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] `M-x checkdoc` is happy with my docstrings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

<!-- After submitting, please fix any problems the CI reports. -->

riscy commented:
Thanks for this.  Quick first pass, mostly centered around the licenses used --

`lurk-mode.el` with [melpazoid](https://github.com/riscy/melpazoid):
```
- lurk-mode.el#L5: Prefer `;; Version` over `;; Package-Version` (MELPA automatically adds `Package-Version`)
```

`ob-lurk.el` with [melpazoid](https://github.com/riscy/melpazoid):
```
- ob-lurk.el#L5: Prefer `;; Version` over `;; Package-Version` (MELPA automatically adds `Package-Version`)
```

License:
- Add *formal* license boilerplate or an [SPDX-License-Identifier](https://spdx.dev/ids/) to the elisp files

weissjeffm commented:
Thanks for the review, @riscy ! I think I have addressed your comments here:
https://github.com/lurk-lang/lurk-emacs/commit/b341ffbf5959bbbc7dd33b35e207ce8b7bfbf565
Let me know if I've missed anything else.

riscy commented:
Thanks for accommodating. And, welcome to MELPA. :)
